src/rust/src/series/general.rs

```diff
@@ -132,4 +132,17 @@ impl PlRSeries {
     fn n_chunks(&self) -> Result<Sexp> {
         (self.series.n_chunks() as i32).try_into()
     }
+
+    fn chunk_lengths(&self) -> Result<Sexp> {
+        let out = self.series.chunk_lengths().collect::<Vec<usize>>();
+        let mut sexp = OwnedIntegerSexp::new(out.len())?;
+        for (i, out) in out.into_iter().enumerate() {
+            let _ = sexp.set_elt(i, out as i32);
+        }
+        Ok(sexp.into())
+    }
+
+    pub fn rechunk(&self) -> Result<PlRSeries> {
+        Ok(self.series.rechunk().into())
+    }
```

Comment:
py-polars has an arg `in_place` but it means that this function returns a `Result<Option<Self>>`: https://github.com/pola-rs/polars/blob/052416c8bf54d6063872d5c3413af9e6845acb78/crates/polars-python/src/series/general.rs#L109

AFAIK, `savvy` cannot return an `Option` (not sure about that), which is why I dropped the argument.

Reply:
Like 031d87182ed574fffe90dabeeb19ee3c3eaa5ec8?

Reply:
Yes, thanks